### PR TITLE
numericIdentifier and textIdentifier passed to the audit log message

### PR DESCRIPTION
**Pull request by rashthedude**
Passing the `numericIdentifier` and `textIdentifier` to the audit log when a user is deleted from an organisation.

### CODE REVIEW

File: src/app/users/removeOrganisationAccess.js [hunk 1]
```diff
@@ -91,7 +94,8 @@ const post = async (req, res) => {
       ],
       editedUser: uid,
     },
-    message: `${req.user.email} (id: ${req.user.sub}) removed organisation ${org} (id: ${organisationId}) for user ${req.session.user.email} (id: ${uid})`,
+    message: `${req.user.email} (id: ${req.user.sub}) removed organisation ${org} (id: ${organisationId}) for user ${req.session.user.email} (id: ${uid})
+      numericIdentifier (${numericIdentifier}) and textIdentifier (${textIdentifier})`,
```
Comment (malikimran34) [inline src/app/users/removeOrganisationAccess.js:98]: make audit logs readable

File: src/app/users/removeOrganisationAccess.js [hunk 2]
```diff
@@ -75,6 +75,9 @@ const post = async (req, res) => {
   const organisationDetails = req.userOrganisations.find((x) => x.organisation.id === organisationId);
   const org = organisationDetails.organisation.name;
 
+  const numericIdentifier = getAllUserDetails.organisations[0]['numericIdentifier'];
+  const textIdentifier = getAllUserDetails.organisations[0]['textIdentifier'];
```
Comment (malikimran34) [inline src/app/users/removeOrganisationAccess.js:79]: does we audit single or multiple org identifier?

File: package.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
     "login.dfe.async-retry": "git+https://github.com/DFE-Digital/login.dfe.async-retry.git#v1.1",
     "login.dfe.audit.transporter": "2.0.2",
     "login.dfe.config.schema.common": "git+https://github.com/DFE-Digital/login.dfe.config.schema.common.git#v1.9.3",
-    "login.dfe.dao": "^2.6.4",
+    "login.dfe.dao": "2.6.4",
```
Comment (malikimran34) [inline package.json:43]: please give ^ a try then npm install

File: src/app/users/removeOrganisationAccess.js [hunk 4]
```diff
@@ -94,8 +98,10 @@ const post = async (req, res) => {
       ],
       editedUser: uid,
     },
-    message: `${req.user.email} (id: ${req.user.sub}) removed organisation ${org} (id: ${organisationId}) for user ${req.session.user.email} (id: ${uid})
-      numericIdentifier (${numericIdentifier}) and textIdentifier (${textIdentifier})`,
+    message: `${req.user.email} (id: ${req.user.sub}) removed organisation ${org} (id: ${organisationId}) for user ${
+      req.session.user.email
+    } (id: ${uid})
+      numeric Identifier and textIdentifier(${JSON.stringify(numericIdentifierAndtextIdentifier)})`,
```
Comment (malikimran34) [inline src/app/users/removeOrganisationAccess.js:104]: typo in textIdentifier it should be 'text identifier'

**Review by malikimran34 — CHANGES_REQUESTED** [review]
can you include the package json and also i am looking into audit database i cannot find any entry just to make sure its logging in the audit table. Thanks

**Comment by rashthedude** [comment]
@malikimran34 I have replaced the audit transporter package which was throwing up errors with Sequelize transport, which seems to be working correctly.

**Review by ipredoi — COMMENTED** [review]
Please check and update the unit tests related to this change in `test/unit/app/users/postRemoveOrganisation.test.js`

**Review by ipredoi — APPROVED** [review]

**Review by malikimran34 — APPROVED** [review]

**Pull request merged**